src/Files.App/ResourceDictionaries/ComboBoxStyle.xaml

```diff
@@ -0,0 +1,9 @@
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+	<Style x:Key="ComboBoxStyle" TargetType="ComboBox">
+		<Setter Property="CornerRadius" Value="4" />
+		<Setter Property="MinWidth" Value="240" />
```

Comment:
What is this based on? I don’t like this kind of hard coding.

Reply:
I tried to dynamically set a width depending on the width of the largest `ComboBoxItem` but I couldn't. In the end I settled with 240, which seemed reasonable during my testing.

Reply:
240 is a bit large, I would aim for 100.

Reply:
Can't we make a dynamic extension to measure the largest item size and set that width as min width as other context menus do?

Reply:
If not, 100 sounds good to me

Reply:
It's better to be dynamic based on the current selected item

Reply:
Could make an extension using dependency object.
100 doesn't have rationale.

Reply:
I understood that as long as a minimum value was provided, the combo box wouldn't shift, is this not the case?

Reply:
Yeah as a workaround this is good enough.

Reply:
@yaira2 @0x5bfa Hey thank you for your suggestions, but 100 seems to be not adequate. Here's a video:

https://github.com/files-community/Files/assets/87632612/5e2e7eb3-e852-47c1-91c1-a31972c9363a

How about a `160`? But to be honest even `160` isn't enough to some long strings

Reply:
I can’t say anything and approve this PR because there’s a much better approach towards this issue; we should set MinWidth set based on the max width of the binding elements using xaml extension.

Reply:
I had originally thought this was a WinUI issue and from this [comment](https://github.com/files-community/Files/issues/14965#issuecomment-1999062570), I thought it meant that setting a value (even if it's `1`), would serve as a workaround. After looking into this further, it seems that this behavior is by design.

It might be best to bring this up on the WinUI repo to learn more about this behavior and what the recommended action is in this case.

Reply:
@yaira2 I found this

https://github.com/microsoft/microsoft-ui-xaml/issues/9009

Reply:
Creating an extension like @0x5bfa suggested sounds like a reasonable approach but it would need to update whenever the selection changes.

Reply:
Yeah but this should've been done by MUX team. Can I work on it?

Reply:
I'm ok with it ig. Thank you for taking it up.

Reply:
Off you can work on it if you want.
Are you gonna work on it with Yair's approval?

Reply:
I think I will leave this to you. I'm still only getting familiar with the codebase and the current solution seems too complex for me atm.

Reply:
What do you think?
I've made a branch and added an extension for that though I don't know if it works yet.

I'll need to add a dependency property inside of DependencyObject-derived class(the extension class) and add `extensions:ComboBoxExtensions.IsKeepWidthEnabled="True"`.

Reply:
@yaira2 here’s the implementation. That easy but seems not working when loaded because ActualWidth is 0 then.

https://github.com/files-community/Files/compare/main...0x5bfa:5bfa/Feature-KeepComboBoxWidth

Reply:
That looks like a neat solution, I think we should move forward with it if that last issue is resolved.